### DISCUSSION TITLE
Project roadmap or something like it

### DISCUSSION THREAD

**Discussion by Danny-dK** [UtrechtUniversity/yoda#190] [Q&A] [answered]
Is there a page or information source somewhere of functionality that is on the map to be implemented in the future (the plans for Yoda and its functionality)? For a basic example, I think you have it on the roadmap to be able to download folders from the webportal, but I don't know whether that is true.

**Comment by stsnel** [comment, marked as answer]
At the moment, the Yoda roadmap is not public yet, but we can share some planning information privately with datamanagers of organizations that use Yoda. We will contact you via email today to send some additional information.
